jax/experimental/jax2tf/tests/correctness_stats.py

```diff
@@ -0,0 +1,136 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import functools
+import numpy as np
+from typing import Callable, List, NamedTuple, Optional, Tuple, Sequence
+
+from jax import core
+from jax import dtypes
+from jax import lax
+from jax import numpy as jnp
+
+def to_jax_dtype(tf_dtype):
+  np_dtype = tf_dtype.as_numpy_dtype
+  # All the TF numpy dtypes come from numpy except for bool, bfloat16.
+  if np_dtype.__module__ == np.__name__:
+    return np_dtype
+  # Only bool has less than 5 elements; in this case, we can still return
+  # np_dtype.
+  elif np_dtype(4) != 4:
+    return np_dtype
+  return jnp.bfloat16
+
+Limitation = NamedTuple("Limitation", [ ("PrimitiveName", str)
```

Comment:
I did not notice this before, but attribute names normally should be lowercase, only type names are capitalized.

Reply:
Thanks, addressed in `09df6e9`.